jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/assignment/RoundRobinAssignmentTest.java

```diff
@@ -174,7 +178,6 @@ public void testMultiActorWithRemovedActor() {
     }
 
     @Test
-    @Ignore("Waits forever")
```

Comment:
As result of this, below import is no longer needed

https://github.com/kiegroup/jbpm/blob/590dcd033067faddb15546761a6d82a7140ec0ab/jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/assignment/RoundRobinAssignmentTest.java#L29